src/api/custom-schema/type-defs.ts

```diff
@@ -248,13 +248,21 @@ const typeDefs = `
     bankAddress: String
   }
 
+  type FinancialInternational {
```

Comment:
`LossPayeeFinancialInternational`?

Reply:
Cannot be `LossPayeeFinancialInternational` as that is an autogenerated one